content/teams/GrowTogether/index.md

```diff
@@ -1,16 +1,16 @@
 ---
 type: "team"                   
-title: DemoTeam
-name: "DemoTeam"
-github-link: "https://github.com/ "
-description: "hello I am Bot Developer"
-Team-Quote: ""
-Team_Tech_Stack: ""
+title: GrowTogether app
+name: "GrowTogether"
```

Comment:
Title and name should be same as name of your team

Reply:
Also in Idea Description : Please Describe your Idea for the project max 4-5 line.

Reply:
Also Give complete github link in github ID section for individual teammate